frontend-react/src/components/Admin/EditSenderSettings.tsx

```diff
@@ -0,0 +1,162 @@
+                    return false;
+            }
+
+            return true;
+        };
+
+        return (
+            <GridContainer>
+                <Grid row>
+                    <Grid col="fill">
+                        Org name:{" "}
+                        {match?.params?.orgname || "missing param 'orgname'"}
+                        <br />
+                        Sender name:{" "}
+                        {match?.params?.sendername ||
+                            "missing param 'sendername'"}
+                        <br />
+                        <br />
+                    </Grid>
+                </Grid>
+                <TextInputComponent
+                    fieldname={"name"}
+                    label={"Name"}
+                    defaultvalue={
+                        action === "edit" ? orgSenderSettings.name : ""
+                    }
+                    savefunc={(v) => (orgSenderSettings.name = v)}
+                    disabled={action === "edit"}
+                />
+                <TextInputComponent
+                    fieldname={"format"}
+                    label={"Format"}
+                    defaultvalue={orgSenderSettings.format}
+                    savefunc={(v) => (orgSenderSettings.format = v)}
+                />
+                <TextInputComponent
+                    fieldname={"topic"}
+                    label={"Topic"}
+                    defaultvalue={orgSenderSettings.topic}
+                    savefunc={(v) => (orgSenderSettings.topic = v)}
+                />
+                <TextInputComponent
+                    fieldname={"customerStatus"}
+                    label={"Customer Status"}
+                    defaultvalue={orgSenderSettings.customerStatus}
+                    savefunc={(v) => (orgSenderSettings.customerStatus = v)}
+                />
+                <TextInputComponent
+                    fieldname={"schemaName"}
+                    label={"Schema Name"}
+                    defaultvalue={orgSenderSettings.schemaName}
+                    savefunc={(v) => (orgSenderSettings.schemaName = v)}
+                />
+                <Grid row>
+                    <Button type="button" onClick={() => history.goBack()}>
+                        Cancel
+                    </Button>
+                    <Button
+                        form="edit-setting"
+                        type="submit"
+                        data-testid="submit"
+                        onClick={() => saveSenderData()}
+                    >
+                        Save
+                    </Button>
+                </Grid>
+            </GridContainer>
+        );
+    };
+
+    return (
+        <NetworkErrorBoundary
+            fallbackComponent={() => <ErrorPage type="page" />}
+        >
+            <Suspense
+                fallback={
+                    <span className="text-normal text-base">
+                        Loading Sender Settings Info...
+                    </span>
```

Comment:
low pri: for all the "Loading..." messages, putting them in a container so they have some left margin is a nice to have

Reply:
@sean-usds fixed